### PR TITLE
Integration update

### PR DESCRIPTION
**Pull request by Tess-LaCoil**
Updated models to remove numerical methods from the von Bertalanffy model and use an analytic solution instead, and to change the Canham method to an rk45 solver built into Stan.

Also made some changes to the provided datasets.

### CODE REVIEW

File: tests/testthat/fixtures/vb/make_vb.R [hunk 1]
```diff
@@ -1,24 +1,24 @@
 #Define specific functions for chosen DE, distribution of parameters, error process
 #Differential equation governing dynamics.
 DE <- function(y, pars){ #Canham function
-  return(pars[1] * (pars[2] - y))
+  return(pars[2] * (pars[1] - y))
```
Comment (dfalster) [inline tests/testthat/fixtures/vb/make_vb.R:4]: Just checking: You sure about swapping the index here?

File: inst/stan/power_single_ind_analytic.stan
Comment (dfalster) [inline inst/stan/power_single_ind_analytic.stan:1]: I though you were removing power law?

File: R/stanmodels.R [hunk 2]
```diff
@@ -11,6 +11,8 @@ Rcpp::loadModule("stan_fit4constant_single_ind_mod", what = TRUE)
 Rcpp::loadModule("stan_fit4linear_single_ind_mod", what = TRUE)
 Rcpp::loadModule("stan_fit4power_multi_ind_mod", what = TRUE)
 Rcpp::loadModule("stan_fit4power_single_ind_mod", what = TRUE)
+Rcpp::loadModule("stan_fit4power_single_ind_analytic_mod", what = TRUE)
```
Comment (dfalster) [inline R/stanmodels.R:14]: I thought you were removing power law?

File: inst/stan/canham_multi_ind.stan [hunk 3]
```diff
@@ -83,22 +42,22 @@ parameters {
...
     if(obs_index[i]==1){//Fits the first size
       y_hat[i] = ind_y_0[ind_id[i]];
     }
 
     if(i < n_obs){
       if(ind_id[i+1]==ind_id[i]){
+        y_temp[1] = y_hat[i];
         //Estimate next size
-        y_hat[i+1] = rk4(y_hat[i], pars, (time[i+1] - time[i]), step_size);
+        y_hat[i+1] = ode_rk45(DE, y_temp,
```
Comment (dfalster) [inline inst/stan/canham_multi_ind.stan:56]: nice!

**Review by dfalster — APPROVED** [review]

File: inst/stan/power_single_ind_analytic.stan
Comment (Tess-LaCoil) [reply inst/stan/power_single_ind_analytic.stan:1]: Yeah I was going to do that separately.

File: tests/testthat/fixtures/vb/make_vb.R (hunk 1 above)
Comment (Tess-LaCoil) [reply tests/testthat/fixtures/vb/make_vb.R:4]: Yes, max size is the first parameter and the growth rate is the second.

**Pull request merged**